### PR TITLE
[Core] Removing some warnings from tests

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

File: kratos/includes/serializer.h [hunk 1]
```diff
@@ -788,8 +789,7 @@ class KRATOS_API(KRATOS_CORE) Serializer
             read(read_tag);
             if(read_tag == rTag)
             {
-                std::cout << "In line " << mNumberOfLines;
-                std::cout << " loading " << rTag << " as expected" << std::endl;
+                KRATOS_INFO("Serializer") << "In line " << mNumberOfLines << " loading " << rTag << " as expected" << std::endl;
```
Comment (philbucher) [inline kratos/includes/serializer.h:792]: :+1:

**Review by philbucher — APPROVED** [review]
awesome, thx!

File: kratos/includes/serializer.h (hunk 1 above)
Comment (loumalouomega) [reply kratos/includes/serializer.h:792]: Let's use a meme

Comment (loumalouomega) [reply kratos/includes/serializer.h:792]: imagen

**Pull request merged**